app/templates/components/events/view/export/api-response.hbs

```diff
@@ -10,22 +10,22 @@
     <UiCheckbox @class="toggle disabled" @label={{t "Basic Event Details"}} @checked={{true}} @onChange={{action "checkboxChange" "basicEventDetails"}} />
   </div>
   <div class="field">
-    <UiCheckbox @class="toggle" @label={{t "Sessions"}} @checked={{this.toggleSwitches.sessions}} @onChange={{action "checkboxChange" "sessions"}} />
+    <LegacyCheckbox @checked={{this.toggleSwitches.sessions}} @label={{t "Sessions"}} @onChange={{action "checkboxChange" "sessions"}}/>
```

Comment:
The only change here should be of name UiCheckbox -> LegacyCheckbox

Please revert all other changes

Reply:
Do we also need the `@class="toggle"` ?

Reply:
The UI of the switch changes according to the class. Same should happen in LegacyCheckbox

Reply:
Then we place the checkbox component and toggle component in the same file as we will be conditionally rendering based on class names. Right?

Reply:
Correct

Reply:
This still has unwanted changes. Once it is reverted, I'll review the PR further

Reply:
@iamareebjamal , updated.